docs/apm/correlations.asciidoc

```diff
@@ -0,0 +1,71 @@
+[role="xpack"]
+[[correlations]]
+=== Find latency and error correlations
+
+**Correlations** surfaces attributes of your data that are potentially correlated with high-latency or erroneous transactions.
+Surfaced attributes are user-defined, meaning that they are completely customizable to your APM data.
+Find something interesting? A quick click of a button will auto-query your data as you work to resolve the underlying issue.
+
+For example, a site reliability engineer, who is responsible for keeping production systems up and running,
+notices an increase in latency in certain transactions.
+Analyzing metadata or tags that exist in high-latency transactions but not in lower-latency transactions
+can potentially point towards the root cause.
+They may find that a particular piece of hardware, like a host or pod, has failed, increasing latency.
+Or, perhaps a set of users, based on IP address or region, is physically too far away from the nearest
+data center, increasing latency.
+
+[discrete]
+[[view-correlations]]
+===  View correlations
+
+With a service selected, click **View correlations**:
+
+[role="screenshot"]
+image::apm/images/correlations.png[Correlations]
+
+Queries within the APM app apply to the correlations shown in the correlations fly-out.
+
+[discrete]
+[[correlations-latency]]
+====  Latency
+
+Correlations help you discover which fields are contributing to increased service latency.
+
+A latency distribution chart visualizes the overall latency of the selected service's transactions.
+Correlated attributes are sorted by _Impact_–a visual representation of the score for the underlying
+aggregation that powers correlations.
+Attributes with a high impact, or attributes present in a large percentage of slow transactions,
+may be contributing factors to increased latency.
+
+If a certain field or value seems noteworthy, use the **Filter** quick links:
+
+* `+` creates a new query in the APM app for transactions containing the selected value.
+* `-` creates a new query in the APM app for transactions without the selected value.
+
+[discrete]
+[[correlations-error-rate]]
+====  Error rate
+
+Correlations help you discover which fields are contributing to failed transactions.
+
+The Error rate over time chart visualizes the change in error rate over the selected time frame.
+Correlated attributes are sorted by _Impact_–a visual representation of the score for the underlying
+aggregation that powers correlations.
+Attributes with a high impact, or attributes present in a large percentage of failed transactions,
+may be contributing factors to increased error rates.
+
+If a certain field or value seems noteworthy, use the **Filter** quick links:
+
+* `+` creates a new query in the APM app for transactions containing the selected value.
+* `-` creates a new query in the APM app for transactions without the selected value.
```

Comment:
For both high-latency and errors, I talk about using the Filter quick links as a next step. Is there more we can give the users here? Where would they go from here from chasing down an abnormal high-latency for example?

Reply:
We expect them to click these filters links to be able to further explore transactions with those terms in more detail elsewhere in APM. They might go to a sample trace to view the logs, error stack traces, etc.

Reply:
Ahhh, I didn't realize that. Done in https://github.com/elastic/kibana/pull/94750.